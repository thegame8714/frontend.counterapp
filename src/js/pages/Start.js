import React, { Component } from "react";
import PageHeading from '../components/PageHeading';

class Start extends Component {
	render() {
		return (
			<div className='content'>
				<PageHeading value="Start page" />
			</div>
		);
	}
}

export default Start;
