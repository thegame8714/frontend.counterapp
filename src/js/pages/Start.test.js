import React from 'react';
import { mount } from 'enzyme';
import { mountToJson } from 'enzyme-to-json';
import Start from './Start';

describe('Start Page', () => {
  it('should render page tile as H1 with text', () => {
    const route = { pageTitle: "Test Start page title" }
    const wrapper = mount(<Start route={route} />);
    expect(mountToJson(wrapper)).toMatchSnapshot();
  });
});