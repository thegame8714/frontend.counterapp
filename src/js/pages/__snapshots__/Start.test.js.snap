exports[`Start Page should render page tile as H1 with text 1`] = `
<Start
  route={
    Object {
      "pageTitle": "Test Start page title",
    }
  }>
  <div
    className="content">
    <PageHeading
      value="Start page">
      <h1>
        Start page
      </h1>
    </PageHeading>
  </div>
</Start>
`;
